server/src/test/java/org/opensearch/index/SegmentReplicationPressureServiceTests.java

```diff
@@ -200,6 +202,7 @@ public void testFailStaleReplicaTask() throws Exception {
             .put(SEGMENT_REPLICATION_INDEXING_PRESSURE_ENABLED.getKey(), true)
             .put(MAX_REPLICATION_TIME_BACKPRESSURE_SETTING.getKey(), TimeValue.timeValueMillis(10))
             .put(MAX_REPLICATION_LIMIT_STALE_REPLICA_SETTING.getKey(), TimeValue.timeValueMillis(20))
+            .put(MAX_INDEXING_CHECKPOINTS.getKey(), 4)
```

Comment:
lets add a test for default 10?

Reply:
IMO, adding a test isn't adding much value . Let me know if you think otherwise .